sdk/communication/communication-short-codes/swagger/README.md

```diff
@@ -17,11 +17,12 @@ model-date-time-as-string: false
 optional-response-headers: true
 payload-flattening-threshold: 10
 use-extension:
-  "@autorest/typescript": "6.0.0-dev.20210223.1"
+  "@autorest/typescript": "6.0.0-beta.15"
 add-credentials: false
 azure-arm: false
 skip-enum-validation: true
 title: Short Codes Client
+use-core-v2: true
```

Comment:
I believe `user-core-v2` is now the default so don't need it here. The wiki might need update.